maxim/logger/livekit/utils.py

```
@@ -149,3 +151,34 @@ def start_new_turn(session_info: SessionStoreEntry):
+        events = llm._events or getattr(llm, "_events", None)
+        if isinstance(events, dict) and "metrics_collected" in events:
+            session_info = get_session_store().get_session_by_agent_session_id(
+                session_id
+            )
+            if session_info is not None:
+                turn = session_info.current_turn
+                if turn is not None:
+                    return turn.usage
+    return None
+
+def extract_llm_model_parameters(llm: Optional[_LLMOptions]):
```

Ruff (0.12.2)

Missing return type annotation for public function `extract_llm_model_parameters`

(ANN201)